crates/universal-sierra-compiler-api/src/lib.rs

```diff
@@ -15,10 +15,16 @@ use shared::command::CommandExt;
 
 mod command;
 
+/// The offset of the sierra statement within the bytecode.
+type CodeOffset = usize;
```

Comment:
This is probably an offset of the casm statement.

Reply:
It is prob the offset in casm bytecode of the first casm statement generated by the sierra statement